test/config.test.js

```diff
@@ -270,6 +270,37 @@ describe('javascript config', () => {
 
 });
 
+
+describe('`logger` config', () => {
+    it('should exist', () => {
+        expect(config.logger).toBeDefined();
+    });
+
+    describe('`dir` it should exist', () => {
```

Comment:
Introducing a different structure into the tests here — I know we discussed this before and agreed that we should hold a discussion on how best to structure tests but can we hold off on changing parts of the test files for now? It's a bit confusing when scanning through the file.

If/when we decide to change this structure I think it should all be done in a single PR, what do you think?

Reply:
Oops, I will update it. I think we should refactor whatever we touch if/when we decide we want to change the structure, instead of doing massive changes in one PR?

Reply:
There aren't that many tests so it might be OK to do the whole lot, depends how much we want to change the structure I guess.